connector/src/main/java/org/geysermc/connector/utils/ItemUtils.java

```diff
@@ -58,4 +58,22 @@ public static int getCorrectBedrockDurability(GeyserSession session, int javaId,
         }
         return original;
     }
+
+    /**
+     * @param itemTag the NBT tag of the item
+     * @return the custom name of the item
+     */
+    public static String getCustomName(CompoundTag itemTag) {
+        if (itemTag != null) {
+            if (itemTag.get("display") instanceof CompoundTag displayTag) {
+                if (displayTag.get("Name") instanceof StringTag nameTag) {
+                    String name = nameTag.getValue();
+                    if (!name.isBlank()) {
```

Comment:
Whats the scenario like if someone sets the custom name to whitespace or zerolength?